### ISSUE TITLE
Astro binding: adding things causes config.xml error regarding lat/lon

### ISSUE THREAD

**Issue by sihui62** [eclipse/smarthome#3727]
Snapshot 961:
Adding an astro thing through PaperUI causes error in openhab.log, also existing things don't trigger anymore:

```
2017-06-23 07:03:51.924 [ERROR] [ding.astro.handler.AstroThingHandler] - Astro parameter geolocation is mandatory and must be configured, disabling thing 'astro:sun:test'
2017-06-23 07:03:51.925 [ERROR] [ding.astro.handler.AstroThingHandler] - Astro parameters geolocation could not be split into latitude and longitude, disabling thing 'astro:sun:test'
2017-06-23 07:03:51.927 [ERROR] [ding.astro.handler.AstroThingHandler] - Astro parameter interval must be in the range of 1-86400, disabling thing 'astro:sun:test'

```
Maybe related to this change: https://github.com/eclipse/smarthome/commit/3585905d04281df5442ce39ea5dfb92611002411

On startup I'm seeing:
```
2017-06-22 15:03:10.439 [WARN ] [ig.xml.osgi.XmlDocumentBundleTracker] - The XML document '/ESH-INF/config/config.xml' in module 'org.eclipse.smarthome.binding.astro' could not be parsed: lat : lat
---- Debugging information ----
message             : lat
cause-exception     : com.thoughtworks.xstream.mapper.CannotResolveClassException
cause-message       : lat
class               : java.util.ArrayList
required-type       : java.util.ArrayList
converter-type      : com.thoughtworks.xstream.converters.collections.CollectionConverter
path                : /config-descriptions/config-description/parameter/description/lat
line number         : 11
class[1]            : org.eclipse.smarthome.config.core.ConfigDescriptionParameter
converter-type[1]   : org.eclipse.smarthome.config.xml.ConfigDescriptionParameterConverter
class[2]            : org.eclipse.smarthome.config.core.ConfigDescription
converter-type[2]   : org.eclipse.smarthome.config.xml.ConfigDescriptionConverter
version             : 1.4.7
-------------------------------
com.thoughtworks.xstream.converters.ConversionException: lat : lat
---- Debugging information ----
message             : lat
cause-exception     : com.thoughtworks.xstream.mapper.CannotResolveClassException
cause-message       : lat
class               : java.util.ArrayList
required-type       : java.util.ArrayList
converter-type      : com.thoughtworks.xstream.converters.collections.CollectionConverter
path                : /config-descriptions/config-description/parameter/description/lat
line number         : 11
class[1]            : org.eclipse.smarthome.config.core.ConfigDescriptionParameter
converter-type[1]   : org.eclipse.smarthome.config.xml.ConfigDescriptionParameterConverter
class[2]            : org.eclipse.smarthome.config.core.ConfigDescription
converter-type[2]   : org.eclipse.smarthome.config.xml.ConfigDescriptionConverter
version             : 1.4.7
-------------------------------
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convert(TreeUnmarshaller.java:79)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.AbstractReferenceUnmarshaller.convert(AbstractReferenceUnmarshaller.java:65)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:66)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:50)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at org.eclipse.smarthome.config.xml.util.ConverterValueMap.readValueMap(ConverterValueMap.java:94)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at org.eclipse.smarthome.config.xml.util.ConverterValueMap.<init>(ConverterValueMap.java:62)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at org.eclipse.smarthome.config.xml.util.ConverterValueMap.<init>(ConverterValueMap.java:41)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at org.eclipse.smarthome.config.xml.ConfigDescriptionParameterConverter.unmarshal(ConfigDescriptionParameterConverter.java:105)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convert(TreeUnmarshaller.java:72)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.AbstractReferenceUnmarshaller.convert(AbstractReferenceUnmarshaller.java:65)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:66)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:50)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readItem(AbstractCollectionConverter.java:71)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.addCurrentElementToCollection(CollectionConverter.java:98)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(CollectionConverter.java:91)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(CollectionConverter.java:85)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.unmarshal(CollectionConverter.java:80)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convert(TreeUnmarshaller.java:72)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.AbstractReferenceUnmarshaller.convert(AbstractReferenceUnmarshaller.java:65)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:66)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:50)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at org.eclipse.smarthome.config.xml.ConfigDescriptionConverter.unmarshal(ConfigDescriptionConverter.java:75)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convert(TreeUnmarshaller.java:72)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.AbstractReferenceUnmarshaller.convert(AbstractReferenceUnmarshaller.java:65)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:66)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:50)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readItem(AbstractCollectionConverter.java:71)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.addCurrentElementToCollection(CollectionConverter.java:98)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(CollectionConverter.java:91)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(CollectionConverter.java:85)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.unmarshal(CollectionConverter.java:80)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convert(TreeUnmarshaller.java:72)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.AbstractReferenceUnmarshaller.convert(AbstractReferenceUnmarshaller.java:65)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:66)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:50)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.start(TreeUnmarshaller.java:134)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.AbstractTreeMarshallingStrategy.unmarshal(AbstractTreeMarshallingStrategy.java:32)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.XStream.unmarshal(XStream.java:1185)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.XStream.unmarshal(XStream.java:1169)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.XStream.fromXML(XStream.java:1115)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.XStream.fromXML(XStream.java:1062)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at org.eclipse.smarthome.config.xml.util.XmlDocumentReader.readFromXML(XmlDocumentReader.java:82)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at org.eclipse.smarthome.config.xml.osgi.XmlDocumentBundleTracker$1.processBundle(XmlDocumentBundleTracker.java:129)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at org.eclipse.smarthome.config.xml.osgi.AbstractAsyncBundleProcessor$1.run(AbstractAsyncBundleProcessor.java:231)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at java.lang.Thread.run(Thread.java:748)[:1.8.0_131]
Caused by: com.thoughtworks.xstream.mapper.CannotResolveClassException: lat
	at com.thoughtworks.xstream.mapper.DefaultMapper.realClass(DefaultMapper.java:79)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.DynamicProxyMapper.realClass(DynamicProxyMapper.java:55)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.PackageAliasingMapper.realClass(PackageAliasingMapper.java:88)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.ClassAliasingMapper.realClass(ClassAliasingMapper.java:79)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.ArrayMapper.realClass(ArrayMapper.java:74)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.SecurityMapper.realClass(SecurityMapper.java:71)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.mapper.CachingMapper.realClass(CachingMapper.java:47)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.util.HierarchicalStreams.readClassType(HierarchicalStreams.java:29)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readItem(AbstractCollectionConverter.java:70)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.addCurrentElementToCollection(CollectionConverter.java:98)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(CollectionConverter.java:91)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(CollectionConverter.java:85)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.unmarshal(CollectionConverter.java:80)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convert(TreeUnmarshaller.java:72)[98:org.eclipse.smarthome.config.xml:0.9.0.201706211154]
	... 44 more
```

Community forum: https://community.openhab.org/t/astro-binding-does-not-trigger-has-no-future-executions-anymore/30403

**Comment by sjka** [comment]
It already has been fixed by #3721

**Comment by kaikreuzer** [comment]
And available since at least distro build 963.
